docs/en/loading/Kafka-connector-starrocks.md

```diff
@@ -11,99 +11,173 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
 - Compared with Routine Load which only supports loading data in CSV, JSON, and Avro formats, Kafka connector can load data in more formats, such as Protobuf. As long as data can be converted into JSON and CSV formats using Kafka Connect's converters, data can be loaded into StarRocks via the Kafka connector.
 - Customize data transformation, such as Debezium-formatted CDC data.
 - Load data from multiple Kafka topics.
-- Load data from Confluent cloud.
+- Load data from Confluent Cloud.
 - Need finer control over load batch sizes, parallelism, and other parameters to achieve a balance between load speed and resource utilization.
 
 ## Preparations
 
 ### Set up Kafka environment
 
-Both self-managed Apache Kafka clusters and Confluent cloud are supported.
+Both self-managed Apache Kafka clusters and Confluent Cloud are supported.
 
-- For a self-managed Apache Kafka cluster, make sure that you deploy the Apache Kafka cluster and Kafka Connect cluster and create topics.
-- For Confluent cloud, make sure that you have a Confluent account and create clusters and topics.
+- For a self-managed Apache Kafka cluster, you can refer to [Apache Kafka quickstart](https://kafka.apache.org/quickstart) to quickly deploy a Kafka cluster. Kafka Connect is already integrated into Kafka.
+- For Confluent Cloud, make sure that you have a Confluent account and have created a cluster.
 
-### Install Kafka connector
+### Download Kafka connector
 
 Submit the Kafka connector into Kafka Connect:
 
 - Self-managed Kafka cluster:
 
-  - Download and unzip [starrocks-kafka-connector](https://github.com/StarRocks/starrocks-connector-for-kafka/releases).
-  - Copy the extracted directory to the path specified in the `plugin.path` property. You can find the `plugin.path` property in the configuration files of worker nodes within the Kafka Connect cluster.
+  Download and extract [starrocks-kafka-connector-xxx.tar.gz](https://github.com/StarRocks/starrocks-connector-for-kafka/releases).
 
-- Confluent cloud:
+- Confluent Cloud:
 
-  > **NOTE**
-  >
-  > The Kafka connector is not currently uploaded to Confluent Hub. You need to upload the compressed file to Confluent cloud.
+  Currently, the Kafka connector is not uploaded to Confluent Hub. You need to download and extract [starrocks-kafka-connector-xxx.tar.gz](https://github.com/StarRocks/starrocks-connector-for-kafka/releases), package it into a zip file and upload the zip file to Confluent Cloud.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/9418aa24-3d0b-4a89-b61d-92af36816516)
1. "xxx" should be a variable that indicates a minor version? If so, I think it's better to the user about that.
2. zip -> .zip (in bold type) or ZIP